src/navigation/context.py

```diff
@@ -111,7 +111,7 @@ def current_gate(self) -> Optional[Gate]:
             post2 = self.get_fid_pos(current_waypoint.fiducial_id + 1)
             if post1 is None or post2 is None:
                 return None
-
+            self.ctx.gate_point_publisher.publish(GPSPointList([convert_cartesian_to_gps(p) for p in [post1, post2]]))
```

Comment:
yea I agree that its a bit awkward, I guess I put it there because I wanted it to publish anytime we have a gate which will always be in this function at that point but I could also put it in the gate state itself I think its unlikely that it would really matter that we wouldn't send it outside of that state

Reply:
yeah I think its fine to put it in gate. the only other time I can think of when we want it to show up is in search state once its found, but since it immediately transitions to gate state, it should only be a couple iterations of delay